__tests__/app.test.js

```diff
@@ -83,4 +55,51 @@ describe("app",()=>{
             })
         })
     })
+    describe("GET /api/articles/:article_id",()=>{
+        test("Status Code: 200 and should return the correct article by id (1)",()=>{
+            return request(app)
+            .get("/api/articles/1")
+            .expect(200)
+            .then(({body})=>{
+                const articleData = body
+                expect(articleData).toMatchObject(
+                    {
+                        title: "Living in the shadow of a great man",
+                        topic: "mitch",
+                    }
```

Comment:
I would also check for the article_id to be 1 - proving you have the correct article